src/mixins/themeable.ts

```diff
@@ -7,22 +7,34 @@ import { ComposeFactory } from '@dojo/compose/compose';
 import { assign } from '@dojo/core/lang';
 
 /**
- * A representation of the css-module class names
- * to be applied where each class in appliedClasses
- * is used.
+ * A representation of the css-module class names to be applied
+ * where each class in appliedClasses is used.
  */
-export type CSSModuleClassNames = {
+export type AppliedCSSModuleClassNames = {
```

Comment:
`AppliedCSSModuleClassName` does not describe the type very well as it can be true or false. Also don't think we need CSSModules in the name (personally)